pkg/model/utxo/snapshot.go

```diff
@@ -23,12 +23,13 @@ func (o *Output) SnapshotBytes() []byte {
 	if err != nil {
 		panic(err)
 	}
+	m.WriteUint32(uint32(len(bytes)))
```

Comment:
It was explicitely added for my mental health :carlos:

I re-added it for several reasons:
- We do write the length before every variable sized object, so its just consistent.
- We would load 32kb from disc and seek back for every single output.
- With future protocol upgrades this could get even worse (bigger block size)

Reply:
> We do write the length before every variable sized object, so its just consistent.

Like what ?